src/table/components/TablePaginationActions.jsx

```diff
@@ -46,11 +75,45 @@ export default function TablePaginationActions(props) {
     }
   };
 
+  const handleSelectPage = (event) => {
+    onPageChange(event, parseInt(event.target.value, 10));
+  };
+
   const onFirstPage = page === 0;
   const onLastPage = page >= Math.ceil(count / rowsPerPage) - 1;
 
+  const options = [];
+  for (let i = 0; i < Math.ceil(count / rowsPerPage); i++) {
+    options.push({ value: i, label: i + 1 });
+  }
+
+  const thinTable = tableWidth < 650;
+
   return (
     <div className={classes.root}>
+      <FormControl className={`${classes.formControl} ${thinTable && classes.hidden}`}>
+        <InputLabel className={classes.caption} htmlFor="pagination-dropdown">
+          {translator.get('SNTable.Pagination.SelectPage')}:{' '}
```

Comment:
```suggestion
          {`${translator.get('SNTable.Pagination.SelectPage')}: `}
```
Does this work? Think syntax is a bit more standard